cookbook/add-tabs.rst

```diff
@@ -0,0 +1,300 @@
+Adding tabs to Sulu admin interface
+===================================
+
+.. note:: It is recommended to read `Extend admin Ui`_ beforehand to get a better understandig of how Sulu admin classes work.
+
+
+This tutorial will walk you through the process of adding extra tabs to the user interface of Sulu.
+
+This could be useful in a few situations, a few examples below:
+
+- Saving extra data for pages
+- Saving extra data in webspaces
+- Saving extra data for users
+
+In this example we'll be adding a "Socials" tab to the PageAdmin of Sulu.
+
+.. figure:: ../img/socials-tab.png
```

Comment:
Can you make a screen shot of the full browser window? Otherwise it looks a little bit weird, because the toolbar is only partly visible. I think you should use a tool like "FireShot" and make the window a bit smaller.